inspect/api.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/forta-network/forta-core-go/domain"
 )
 
+const (
+	blockByNumber = "eth_getBlockByNumber"
+	// TODO: change this to safe or finalized after merge.
+	safeHead = "latest"
```

Comment:
once we set this to `safe` after merge, only the upgraded nodes should pass @stevenlanders